.github/CODEOWNERS

```diff
@@ -0,0 +1,21 @@
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# @cabinetoffice-org/nlc-deployment team will be requested
+# for review when someone opens a pull request.
+*       @cabinetoffice-org/nlc-deployment
+
+# When someone opens a pull request that modifies files in
+# /web
+# /config
+# /scripts
+# etc.
+# only @joesb and not the global owner(s) will be requested
+# for a review.
+/web @joesb
+/config @joesb
+/scripts @joesb
+/app_cron @joesb
+/console @joesb
+composer.* @joesb
+
+/docker-build @mikebell @stutily
```

Comment:
In the branch for aws changes this is renamed to /deployment/ - in fact we will need to merge that to master before we do this merge I think ...

Reply:
I've added a line for this in commit at ef67245